src/java/org/apache/ivy/plugins/resolver/IBiblioResolver.java

```diff
@@ -191,7 +191,34 @@ public void endElement(String uri, String localName, String qName)
                         
                         return rev;
                     }
-                } else {
+                } else if (root.startsWith("file:")) {
+                    // sbt change, driven by https://github.com/sbt/sbt/issues/1616
+                    String metadataLocalLocation = IvyPatternHelper.substitute(
+                    root + "[organisation]/[module]/[revision]/maven-metadata-local.xml", mrid);
+                    Resource metadataLocal = getRepository().getResource(metadataLocalLocation);
+                    if (metadataLocal.exists()) {
+                        Message.verbose("maven-metadata-local found: " + metadataLocal);
+                        metadataStream = metadataLocal.openStream();
+                        final StringBuffer localCopy = new StringBuffer();
```

Comment:
I'm using `StringBuffer` since the code right above is using it too. `StringBuffer` is synchronized, but not sure if that's needed.